Units/review-needed.r/simple.js.t/expected.tags

```diff
@@ -1,5 +1,5 @@
 Database	input.js	/^Database.prototype.getTodaysDate = Database_getTodaysDate;$/;"	c
-ValidClassOne	input.js	/^testlib.extras.ValidClassOne = function(a,b) { $/;"	c	class:testlib.extras
+ValidClassOne	input.js	/^testlib.extras.ValidClassOne = function(a,b) { $/;"	c	scope:testlib.extras
```

Comment:
Very interesting.
I have many comments about this line.
...

Reply:
Yes, this is a tricky issue.  The parser doesn't always know what the thing is.  Perhaps the cork queue will help solve this?